reactive/flannel.py

```diff
@@ -223,9 +223,33 @@ def reset_states_and_redeploy():
 def cleanup_deployment():
     ''' Terminate services, and remove the deployed bins '''
 
+
     host.service_stop('flannel')
+
+    # Stop and remove the flannel bridge
+    down = ['ip', 'link', 'set', 'flannel.1', 'down']
+    delete = ['ip', 'link', 'delete', 'flannel.1']
+    try:
+        subprocess.check_call(down)
+        subprocess.check_call(delete)
+    except subprocess.CalledProcessError:
```

Comment:
If we're going to ignore this exception here, can we at least log it for troubleshooting purposes?